### PR TITLE
Dylibso.Extism.CLI version 1.6.1

### PR DESCRIPTION
**Pull request by ero-qt**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/195378)

### CODE REVIEW

**Comment by just-ero** [comment]
@microsoft-github-policy-service agree

**Comment by just-ero** [comment]
I'm not entirely sure what the problem is?

File: manifests/d/Dylibso/Extism/CLI/1.6.1/Dylibso.Extism.CLI.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+License: BSD-3-Clause License
+LicenseUrl: https://github.com/extism/cli/blob/main/LICENSE
+ShortDescription: Extism CLI generates and executes Extism plugins, and manages libextism installations.
+Tags:
+- wasm
+- wasi
+ReleaseNotes: |
+  Commits
+  - feat: support enable-http-response-headers flag, minor tweaks
+Icons:
```
Comment (DuckDuckStudio) [inline manifests/d/Dylibso/Extism/CLI/1.6.1/Dylibso.Extism.CLI.locale.en-US.yaml:21]:
```
Manifest Warning: Field usage requires verified publishers. [Icons]
```

Try removing [Icons] from the manifest.

File: manifests/d/Dylibso/Extism/CLI/1.6.1/Dylibso.Extism.CLI.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- IconUrl: https://avatars.githubusercontent.com/u/84687501
+  IconFileType: png
+  IconResolution: 256x256
+  IconTheme: default
```
Comment (DuckDuckStudio) [inline manifests/d/Dylibso/Extism/CLI/1.6.1/Dylibso.Extism.CLI.locale.en-US.yaml:25]:
```
Manifest Warning: Field usage requires verified publishers. [Icons]
```

Try removing [Icons] from the manifest.

Comment (mdanish-kh) [reply manifests/d/Dylibso/Extism/CLI/1.6.1/Dylibso.Extism.CLI.locale.en-US.yaml:25]: Icons are not accepted in the community repository. This should be removed from the manifest

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/195378/files#r1864230918

**Comment by just-ero** [comment]
Do I need to restart the pipeline run manually?

**Review by mdanish-kh — APPROVED** [review]

**Review by DuckDuckStudio — APPROVED** [review]

**Pull request merged**